src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -2622,13 +2622,15 @@ void InitSolarHeatGains(EnergyPlusData &state)
         Real64 GndReflSolarRad = 0.0;
         Real64 GndSolarRadInc = max(state.dataEnvrn->BeamSolarRad * state.dataEnvrn->SOLCOS(3) + state.dataEnvrn->DifSolarRad, 0.0);
         for (int SurfNum = 1; SurfNum <= state.dataSurface->TotSurfaces; ++SurfNum) {
-            state.dataSurface->SurfSkySolarInc(SurfNum) = state.dataEnvrn->DifSolarRad * state.dataSolarShading->SurfAnisoSkyMult(SurfNum);
+            Real64 SurfIncSolarMultiplier = GetSurfIncidentSolarMultiplier(state, SurfNum);
```

Comment:
Is the multiplier 1.0 for all surfaces other than exterior windows? In that case, you probably don't want to call this function for every surface. What you want to do is have an array initialized to 1.0 for every surface and then at the beginning of every time step, call this function only for exterior windows and overwrite their entries with the scheduled value.

Reply:
> Is the multiplier 1.0 for all surfaces other than exterior windows?
Yes. That's the intention. Only for exterior surfaces with user-defined incidentSolarMultiplier object can equal to values other than 1.0. 

I will rearrange the code and overwrite the values for exterior window instead of calling this function on all surfaces.